### PR TITLE
[WIP] Add suspense fixtures for IO and CPU demo

### PR DESCRIPTION
**Pull request by bvaughn**
- [x] Imported the demo apps from [Dan's JSConf Iceland 2018](https://reactjs.org/blog/2018/03/01/sneak-peek-beyond-react-16.html)
- [x] Updated them to work with suspense in master
- [x] Add basic `README.md` files with instructions on how to run
- [x] Fix or disable CPU "async" option; it does not currently seem to be time slicing work correctly
- [x] Replace Rotten Tomatoes mock UI with other data

### CODE REVIEW

File: fixtures/suspense/io/src/components/MovieListPage.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+// --------------------------
+// Movie list page
+// --------------------------
+//
+// Top Box Office
+// - 🍅 97% Black Panther
+// - 🤢 58% Peter Rabbit
+// - 🤢 12% Fifty Shades Freed
+// --------------------------
```
Comment (nhunzaker) [inline fixtures/suspense/io/src/components/MovieListPage.js]: Definitely

File: fixtures/suspense/io/src/index.css [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+.🕹:hover {
+  background-color: white;
+  top: -4px;
+  left: -4px;
+  box-shadow: 4px 4px 0 var(--color-buttonBorder);
+}
+
+.🕹:active {
```
Comment (nhunzaker) [inline fixtures/suspense/io/src/index.css]: TIL you can use emoji as class names. I guess they're just characters.

Comment (nhunzaker) [reply fixtures/suspense/io/src/index.css]: That would be a joke if the emoji movie were in here.

**Comment by bvaughn** [comment]
Thanks for the thorough review, everyone.

Assuming CI is happy, I'm going to land this.

**Comment by gaearon** [comment]
Is the CPU thing still an issue? I can look now.

**Comment by bvaughn** [comment]
Whoops, your comment popped in after I clicked "merge"

Yes, I disabled the "async" button for the CPU demo for now– because it didn't seem to be working correctly. I assume it has to do with lack of support fo resuming.

**Comment by gaearon** [comment]
I wouldn't expect resuming to be related (last time we didn't have it either). Will look.

**Pull request merged**